package.nls.json

```diff
@@ -85,6 +85,9 @@
     "AWS.message.prompt.credentials.definition.help": "Would you like some information related to defining credentials?",
     "AWS.message.prompt.credentials.definition.tryAgain": "The credentials do not appear to be valid ({0}). Would you like to try again?",
     "AWS.message.prompt.selectLocalLambda.placeholder": "Select a lambda function",
+    "AWS.message.prompt.welcome.toastMessage": "You are now using the AWS Toolkit for Visual Studio Code, version {0}",
+    "AWS.message.prompt.welcome.openPage": "View the Welcome Page",
```

Comment:
Nit: What do you think of "View Welcome Page"?

Reply:
✔️